src/main/java/uk/gov/dvsa/model/cvs/MSVA30.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.dvsa.model.cvs;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.dvsa.model.Document;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class MSVA30 extends Document {
+    @JsonProperty("dummyProperty")
+    private String dummyProperty;
+
+    public String getDummyProperty() {
```

Comment:
if these are dummy properties, is the getter and setter needed?

Reply:
Only present until other work is done